### DISCUSSION TITLE
TaxonomyPart is missing

### DISCUSSION THREAD

**Discussion by smemis** [OrchardCMS/OrchardCore#13554] [General]
/Admin/ContentParts/Edit/TaxonomyPart
The page could not be found.

OrchardCore.Taxonomies enabled.

**Comment by hishamco** [comment]
It's there, whatt verrsion you are using?

![Taxonomies](https://user-images.githubusercontent.com/3237266/231564589-6fed2168-7ce6-40f4-b478-995be7147b84.png)

**Comment by smemis** [comment]
OC 1.3.0 and OC 1.6.0-preview-17638

<img width="1916" alt="image" src="https://user-images.githubusercontent.com/4637285/231597188-c2fc1d69-d942-4dcc-aac5-b055ad7abe23.png">

**Reply by hishamco** [reply]
If you mean it's not there in the parts view, I notice that also there's a `Taxonomy` content type that should be there. I'm not when the change was made, @MikeAlhayek any idea

**Reply by MikeAlhayek** [reply]
We do not expose `TaxonomyPart` to the part definitions. So it is not something you can add to your content type. If you want Taxonomy, you should add TaxonomyField to one of your parts.

1. Taxonomy content item. 
2. Add a TaxonomyField to your custom ContentType via a part. 
3. Edit the TaxonomyField settings and select the content item you created in step 1.

**Comment by hishamco** [comment]
You're right, I forgot about `TaxonomyField` but AFAIK all the parts are listed on the content parts, are there any parts don't show up?

**Reply by MikeAlhayek** [reply]
In order for the part to show up, it needs to be defined in the content definition. TexonomyPart is registered in the OC but it is not defined in the content definitions so it won't show up. Only attachable parts will show up.

**Comment by hishamco** [comment]
You're right, I forgot about `TaxonomyField` but AFAIK all the parts are listed on the content parts, are there any parts don't show up?
